packages/delivery-utils/src/array-from-node-list.ts

```diff
@@ -0,0 +1,7 @@
+export function arrayFromNodeList(input: NodeList): HTMLElement[] {
```

Comment:
Not sure what all of your use cases are, but the native method `Array.from()` will perform this same operation. One exception being that IE doesn't support `Array.from()`. I couldn't tell you off the top of my head if the use of `Array.prototype.slice()` in this fashion will work in IE.

I made a pared down shim for `Array.from()` a while back, which omits a lot of the irrelevant code in the [official polyfill](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from#polyfill). It uses `forEach()` instead of `Array.prototype.slice()`. It also handles maps and sets.

https://github.com/SentientTechnologies/execution-plan/blob/master/src/shared/shims/from-array.shim.ts

Reply:
The TypeScript ponyfill for [spreading arrays](https://github.com/microsoft/tslib/blob/481d35218af580ec37a6a8f6b836b80cb1a245b3/tslib.js#L207-L215) also uses `Array.prototype.slice()`, so you're probably safe for IE compatibility